libraries/abstractions/pkcs11/mbedtls/iot_pkcs11_mbedtls.c

```diff
@@ -114,22 +114,74 @@ static const char * pNoLowLevelMbedTlsCodeStr = "<No-Low-Level-Code>";
  * @brief Indicates that no PKCS #11 operation is underway for given session.
  *
  */
-#define pkcs11NO_OPERATION     ( ( CK_MECHANISM_TYPE ) -1 )
+#define pkcs11NO_OPERATION                  ( ( CK_MECHANISM_TYPE ) -1 )
+
+/**
+ * @ingroup pkcs11_macros
+ * @brief size of a prime256v1 EC private key in bytes, when encoded in DER.
+ *
+ *
+ */
+#define pkcs11_PRIVATE_EC_PRIME_256_SIZE    160
+
+/**
+ * @ingroup pkcs11_macros
+ * @brief size of a prime256v1 EC public key in bytes, when encoded in DER.
+ *
+ */
+#define pkcs11_PUBLIC_EC_PRIME_256_SIZE     100
+
+/**
+ * @ingroup pkcs11_macros
+ * @brief size of a 2048 bit RSA public key in bytes, when encoded in DER.
+ *
+ *
+ */
+#define pkcs11_PUBLIC_RSA_2048_SIZE         300
+
+/**
+ * @ingroup pkcs11_macros
+ * @brief size of a 2048 bit RSA private key in bytes, in DER encoding.
+ *
+ */
+#define pkcs11_PRIVATE_RSA_2048_SIZE        1200
+
+/**
+ * @ingroup pkcs11_macros
+ * @brief Max size of an EC public key in bytes, in DER encoding.
+ */
+#define EC_MAX_PUBLIC_KEY_SIZE              pkcs11_PUBLIC_EC_PRIME_256_SIZE
```

Comment:
We can make the next 4 macros consistent in naming (using size as opposed to Length):

```c
#define MAX_EC_PUBLIC_KEY_DER_SIZE              pkcs11_EC_PRIME_256_PUBLIC_KEY_DER_SIZE
#define MAX_EC_PRIVATE_KEY_DER_SIZE      pkcs11_EC_PRIME_256_PRIVATE_KEY_DER_SIZE
#define MAX_PUBLIC_KEY_DER_SIZE    pkcs11_RSA_2048_PUBLIC_KEY_DER_SIZE
#define MAX_PRIVATE_KEY_DER_SIZE    pkcs11_RSA_2048_PRIVATE_KEY_DER_SIZE
```

Reply:
Done